recipe/conda_build_config.yaml

```diff
@@ -210,23 +210,23 @@ boost:
 boost_cpp:
   - 1.66.0
 bzip2:
-  - 1.0
+  - 1
 cairo:
   - 1.14
 curl:
-  - 7.44
+  - 7
```

Comment:
Should we still keep these as they determine what the build environment uses version-wise? Little worried we might have trouble keeping the stack consistent if we don't keep the lower bound pinned tightly across the board.

Reply:
This means that if `7.45` was used, then we'd have `>=7.45` in run requirements. Why would this break the consistency of the stack?

Reply:
Because the lower bound will drift upwards over time as `curl` (in this case) is updated. So the range will change for packages built with newer versions of `curl`.

Reply:
Right, but since the versions are compatible, isn't that okay?

Reply:
It just makes it harder for people to know what the range of supported `curl`'s (or other packages') versions in conda-forge. This may also make it harder for users to install specific versions of `curl` (or other packages') should they have other constraints in their deployment environment. Keeping the lower bound fixed avoids this issue.

Reply:
I get your point, but the problem is that we are already requiring the latest version. For example in libpng, we had this constraint `libpng >=1.6.32,<1.6.35`, but we can't use a package built with 1.6.34 with 1.6.32 on OSX, because the compatibility-version requirement is wrong. I'm pretty sure this will be the same for curl. What you are proposing is fine when rebuilding pacakges, but at the moment, it's not possible because packages actually require newer versions than `1.6.32` even though the requirement is  `libpng >=1.6.32,<1.6.35`

Reply:
Sure. So I think it would be pretty defensible for us to bump the lower bounds to something that works with the existing stack, but side steps these known compatibility issue. For instance, in the case of `libpng`, what about using 1.6.34 as the lower bound (at least for the near term)?

Reply:
That works, but for `curl`, you'll have to bump it every time you do a minor release update to avoid breakage with old style pinnings with the new version.

Reply:
Is that still true if we keep `max_pin: x` for `curl`? Or are there some issues with that approach?

Reply:
Yeah, we'll be building with 7.59 which use newer pinnings and we'll be building with 7.60 in the future when building with older pinnings.

Reply:
@isuruf @jakirkham please note that the oldest `curl` version available on anaconda cloud is 7.48 (https://anaconda.org/conda-forge/curl/files). In fact, unpinned curl in builds errors out, see this [Appveyor log](https://ci.appveyor.com/project/conda-forge/ffmpeg-feedstock/build/1.0.80/job/c49e75ur451d47nd) from https://github.com/conda-forge/ffmpeg-feedstock/pull/47

Reply:
Making this `7.48` seems fine.

Reply:
This particular point appears to be solved with PR ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/55 ). Would be good to extend the same sort of solution to the rest of the pins.